llvm/test/tools/llvm-objcopy/ELF/compress-sections-within-segment.s

```diff
@@ -30,3 +35,24 @@ Sections:
     Type:     SHT_PROGBITS
     Flags:    [ SHF_ALLOC, SHF_COMPRESSED ]
     Content:  010000000000000040000000000000000100000000000000789cd36280002d3269002f800151
+
+--- !ELF
+FileHeader:
+  Class:      ELFCLASS64
+  Data:       ELFDATA2LSB
+  Type:       ET_EXEC
+  Machine:    EM_X86_64
+ProgramHeaders:
+  - Type:     PT_LOAD
+    FirstSec: foo
+    LastSec:  foo
+    VAddr:    0x201000
+    Align:    0x1000
+    Offset:   0x1000
+Sections:
+  - Name:     foo
```

Comment:
In the first YAML, you've got both `foo` and `.text`, but here you only have `foo`. Would it be simpler to have both cases use the same YAML, but simply change the llvm-objcopy options for the two runs? That would have the added benefit that in this second case, it shows that uncompressed sections that are not being compressed don't result in an error if they are in a segment.

Reply:
Removed addresses